PendingReleaseNotes.md

```diff
@@ -8,3 +8,8 @@
 
 - The toolbox pod now uses the Ceph image directly instead of the Rook image
 - Add support for AWS Server Side Encryption with (AWS-SSE:S3)[https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html] for RGW.
+
+## Documentation
```

Comment:
> I think you can remove changes from the pending release notes. We usually don't flag doc updates there anyway.

Agreed, no need for the release notes, this is just for new 1.10 features. Anyway, we will backport to the 1.9 docs instead of waiting for the 1.10 release.

Reply:
- [ ] remove pending release notes changes